zombienet/spin/spin.toml

```diff
@@ -1,5 +1,5 @@
 [relaychain]
-default_command = "./target/debug/qf-node"
```

Comment:
This is the default path that you are most likely to find on every developer's machine. Can you resolve it for your setup with a symlink like `ln -s ~/.cargo/target ./target`?